crates/ekore/src/anomalous_dimensions/unpolarized/spacelike/as4/ggg.rs

```diff
@@ -0,0 +1,201 @@
+/// Compute the singlet gluon-to-gluon anomalous dimension.
+use num::complex::Complex;
+use num::traits::Pow;
+
+use crate::constants::{ZETA2, ZETA3};
+use crate::harmonics::cache::{Cache, K};
+use crate::harmonics::log_functions::{
+    lm11, lm11m1, lm11m2, lm12m1, lm12m2, lm13m1, lm13m2, lm14m1,
+};
+
+// The routine is taken from [\[Falcioni:2024qpd\]][crate::bib:Falcioni:2024qpd].
+pub fn gamma_gg(c: &mut Cache, nf: u8, variation: u8) -> Complex<f64> {
+    let n = c.n();
+    let S1 = c.get(K::S1);
+    let S2 = c.get(K::S2);
+    let S3 = c.get(K::S3);
+    let S4 = c.get(K::S4);
+    let nf2 = nf.pow(2) as f64;
```

Comment:
should we also cast `nf` here once and for all? (note that we want `u8` as input, and only cast internally)

Reply:
done in f8eb85ce

Reply:
Improved in e91e0bda :upside_down_face: